<launch>
    <!-- add Signal Map -->

    <node pkg="occgrid_planner" type="signal_grid_builder" name="signal_grid_builder" output="screen">
        <param name="base_frame" value="/body"/>
        <param name="sg_grid" value="/map"/>
        <param name="mapSizeX" value="32" type="int"/>
        <param name="mapSizeY" value="32" type="int"/>
        <param name="resolution" value="1.0"/>
        <param name="maxSignalValue" value="0.5"/>
        <remap from="/SignalGridBuilder/signal" to="/vrep/metalDetector"/>
        <remap from="/SignalGridBuilder/signalMap" to="/SignalGridBuilder/signalMap"/>
    </node>
</launch>